main-packages-list.ocp

```diff
@@ -43,6 +43,7 @@ python3-packaging >= 20.4-1.el8ost
 python3-paste >= 3.2.4-1.el8ost
 python3-paste-deploy >= 2.0.1-4.el8ost
 python3-pint >= 0.10.1-1.el8ost
+python3-proliantutils
```

Comment:
This probably should have been added to the OKD pkgs list as well?

Reply:
++ we really need an okd job so that we stop doing this mistake

Reply:
d'oh! sorry, you're right :/